File: Client/lib/features/updater/models/github_asset_info.dart
class GithubAssetInfo {
  final String name;
  final String contentType;
  final String createdAt;
  final String publishedAt;
  final String browserDownloadUrl;

  const GithubAssetInfo(
    this.name,
    this.contentType,
    this.createdAt,
    this.publishedAt,
    this.browserDownloadUrl,
  );

  factory GithubAssetInfo.fromJson(Map<String, dynamic> json) {
    return GithubAssetInfo(
      json['name'] ?? '',
      json['content_type'] ?? '',
      json['created_at'] ?? '',
      json['updated_at'] ?? '',
      json['browser_download_url'] ?? '',
    );
  }
}

File: Desktop/Features/Updater/Models/GithubAssetInfo.cs
using System.Text.Json.Serialization;

namespace StreamTabula.Features.Updater.Models
{
    public class GithubAssetInfo()
    {
        [JsonPropertyName("name")]
        public string Name { get; set; } = string.Empty;

        [JsonPropertyName("content_type")]
        public string ContentType { get; set; } = string.Empty;

        [JsonPropertyName("created_at")]
        public string CreatedAt { get; set; } = string.Empty;

        [JsonPropertyName("published_at")]
        public string PublishedAt { get; set; } = string.Empty;

        [JsonPropertyName("browser_download_url")]
        public string BrowserDownloadUrl { get; set; } = string.Empty;
    }
}